CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Catalog: Connector filter is no longer split into ID and endpoint
```

Comment:
![grafik](https://github.com/user-attachments/assets/ce76cf4c-ffc4-4a4d-9fcb-f9f46c5e99c3)

On submit it does not switch to the third page